### ISSUE TITLE
Project Dead?

### ISSUE THREAD

**Issue by MurmeltierS** [davidkpiano/flipping#41]
This project seems very dead, are there any viable alternatives?

**Comment by davidkpiano** [comment]
Yes, this is becoming a native feature: https://www.youtube.com/watch?v=JCJUPJ_zDQ4

**Comment by ilyasmez** [comment]
> Yes, this is becoming a native feature: https://www.youtube.com/watch?v=JCJUPJ_zDQ4

The new View Transitions API will cover only the view transitions use case, while this library can be used for many other use cases (e.g. running performant animation in _the same_ page)

**Comment by MurmeltierS** [comment]
@ilyasmez I would suggest taking a look at https://auto-animate.formkit.com/

**Comment by Char2sGu** [comment]
A more advanced animation technique that follows a similar idea would be Layout Projection.
See https://github.com/Char2sGu/layout-projection for more info
